src/time_evolution/ssesolve.jl

```diff
@@ -0,0 +1,407 @@
+        H_t::Union{Nothing,Function,TimeDependentOperatorSum}=nothing,
+        params::NamedTuple=NamedTuple(),
+        n_traj::Int=1,
+        ensemble_method=EnsembleThreads(),
+        prob_func::Function=_mcsolve_prob_func,
+        output_func::Function=_mcsolve_output_func,
+        kwargs...)
+
+
+Stochastic Schrödinger equation evolution of a quantum system given the system Hamiltonian ``\hat{H}`` and a list of stochadtic collapse (jump) operators ``\{\hat{C}_n\}_n``.
+The stochastic evolution of the state ``|\psi(t)\rangle`` is defined by:
+    
+    ```math
+    d|\psi(t)\rangle = -i K |\psi(t)\rangle dt + \sum_n M_n |\psi(t)\rangle dW_n(t)
+    ```
+
+where 
+    
+```math
+    K = \hat{H} + i \sum_n \left(\frac{e_j} C_n - \frac{1}{2} \sum_{j} C_n^\dagger C_n - \frac{e_j^2}{8}\right),
+    ```
+    ```math
+    M_n = C_n - \frac{e_n}{2},
+    ```
+    and
+    ```math
+    e_n = \langle C_n + C_n^\dagger \rangle.
+    ```
+
+Above, `C_n` is the `n`-th collapse operator and  `dW_j(t)` is the real Wiener increment associated to `C_n`.
+
+
+# Arguments
+
+- `H::QuantumObject`: Hamiltonian of the system ``\hat{H}``.
+- `ψ0::QuantumObject`: Initial state of the system ``|\psi(0)\rangle``.
+- `tlist::AbstractVector`: List of times at which to save the state of the system.
+- `sc_ops::Vector`: List of stochastic collapse operators ``\{\hat{C}_n\}_n``.
+- `alg::StochasticDiffEq.StochasticDiffEqAlgorithm`: Algorithm to use for the time evolution.
+- `e_ops::Vector`: List of operators for which to calculate expectation values.
+- `H_t::Union{Nothing,Function,TimeDependentOperatorSum}`: Time-dependent part of the Hamiltonian.
+- `params::NamedTuple`: Dictionary of parameters to pass to the solver.
+- `seeds::Union{Nothing, Vector{Int}}`: List of seeds for the random number generator. Length must be equal to the number of trajectories provided.
+- `n_traj::Int`: Number of trajectories to use.
+- `ensemble_method`: Ensemble method to use.
+- `prob_func::Function`: Function to use for generating the SDEProblem.
+- `output_func::Function`: Function to use for generating the output of a single trajectory.
+- `kwargs...`: Additional keyword arguments to pass to the solver.
+
+# Notes
+
+- `ensemble_method` can be one of `EnsembleThreads()`, `EnsembleSerial()`, `EnsembleDistributed()`
+- The states will be saved depend on the keyword argument `saveat` in `kwargs`.
+- If `e_ops` is specified, the default value of `saveat=[tlist[end]]` (only save the final state), otherwise, `saveat=tlist` (saving the states corresponding to `tlist`). You can also specify `e_ops` and `saveat` separately.
+- The default tolerances in `kwargs` are given as `reltol=1e-6` and `abstol=1e-8`.
+- For more details about `alg` and extra `kwargs`, please refer to [`DifferentialEquations.jl`](https://diffeq.sciml.ai/stable/)
+
+# Returns
+
+- `sol::TimeEvolutionSSESol`: The solution of the time evolution. See also [`TimeEvolutionSSESol`](@ref)
+"""
+
```

Comment:
There is and extra line here, and so the documentation failed.

Reply:
I actually think it's because TimeEvolutionSSESol was not being exported and because it was missing the docs. Im adding them now

Reply:
Yes, this is one, but now you have to put it into the documentation, otherwise we have an error. 

Moreover, what I was saying is that the documentation failed because the string should be exactly above the respective code. Indeed, the returned error is
```julia-repl
Error: no docs found for 'ssesolve' in `@docs` block in src/api.md:175-198
```